bin/run.sh

```diff
@@ -0,0 +1,216 @@
+#!/bin/bash -x
```

Comment:
Please let's name this run-v2v.sh so we can distinguish between v2v and url scripts

Reply:
OK